staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -927,7 +927,8 @@ func (e *Store) updateForGracefulDeletionAndFinalizers(ctx context.Context, name
 			if !graceful {
 				// set the DeleteGracePeriods to 0 if the object has pendingFinalizers but not supporting graceful deletion
 				if pendingFinalizers {
-					klog.V(6).Infof("update the DeletionTimestamp to \"now\" and GracePeriodSeconds to 0 for object %s, because it has pending finalizers", name)
+					klog.V(6).InfoS("Object has pending finalizers, so the registry is going to update its status to deleting",
+						"object", klog.KRef(genericapirequest.NamespaceValue(ctx), name), "DeletionTimestamp", "\"now\"", "GracePeriodSeconds", "0")
```

Comment:
Not sure if we should pass hardcoded arguments into 
We shouldn't pass "now" into "deletionTimestamp" as it is not correct timestamp type. Depending on what "now" means we should either pass `time.now()` or not extract key at all.

As for grace period we should not do serialization ourselfs but pass correct time difference type.

```suggestion
						"object", klog.KRef(genericapirequest.NamespaceValue(ctx), name), "gracePeriod", time.Second * 0)
```